python/rascaline/examples/le-basis.py

```diff
@@ -0,0 +1,197 @@
+"""
+.. _example-le-basis:
+
+Laplacian eigenstate basis
+==========================
+
+.. start-body
+"""
+
+# %%
+#
+# This script illustrates how to generate a spherical expansion using
+# the Laplacian eigenstate basis (https://doi.org/10.1063/5.0124363),
+# both using truncation with l_max, n_max hyper-parameters and with an
+# eigenvalue threshold.
+
+import ase.io
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.optimize
+from metatensor import Labels, TensorBlock, TensorMap
+from scipy.special import jv
+
+import rascaline
+
+
+# %%
+#
+# First using a truncation with l_max, n_max hyper-parameters (easy):
+cutoff = 4.4
+l_max = 6
+n_max = 8
+
+structures = ase.io.read("dataset.xyz", ":10")
+
+spliner = rascaline.utils.SoapSpliner(
+    cutoff=cutoff,
+    max_radial=n_max,
+    max_angular=l_max,
+    basis=rascaline.utils.SphericalBesselBasis(
+        cutoff=cutoff, max_radial=n_max, max_angular=l_max
+    ),
+    density=rascaline.utils.DeltaDensity(),
+    accuracy=1e-8,
+)
+
+calculator = rascaline.SphericalExpansion(
+    cutoff=cutoff,
+    max_radial=n_max,
+    max_angular=l_max,
+    center_atom_weight=1.0,
+    radial_basis=spliner.compute(),
+    atomic_gaussian_width=-1.0,  # will not be used due to the delta density above
+    cutoff_function={"ShiftedCosine": {"width": 0.5}},
+)
+
+spherical_expansion = calculator.compute(structures)
+
+# %%
+#
+# Now we will calculate the same basis with an eigenvalue threshold
+# (more involved), which affords a better accuracy/cost ratio, using
+# property selection.
+
+E_max = 400  # eigenvalue threshold
+
+# %%
+#
+# Spherical Bessel zeros (from the scipy cookbook) and
```

Comment:
Is there a direct link to the cookbook? If yes we should add this here, to the other example and to the docs of `SphericalBesselBasis`.

Also while I am thinking about this. We now have the same lines of code in three places. Does it make sense to define these functions within rascaline directly?

Reply:
True, I'll try to add a member function to `SphericalBesselBasis`